dockstore-client/generated/src/main/resources/pom.xml

```diff
@@ -342,6 +342,36 @@
       <version>1.3.5</version>
       <scope>compile</scope>
     </dependency>
+    <dependency>
```

Comment:
I suspect that not all of these are needed, as discussed, it might be because of the granularity of the code we're re-using (i.e. may need to split out some of the re-usable code)

Reply:
> 
In regards to `AbstractWorkflowResource.java` it appears that some validation of the Dockstore Yaml is being done there.
https://github.com/dockstore/dockstore/blob/3680542eda82ec9728418b6a8237e591936c484e/dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractWorkflowResource.java#L364

Validation that is being done in  `AbstractWorkflowResource.java` includes [verifying that WDL is not being used as a subclass](https://github.com/dockstore/dockstore/blob/3680542eda82ec9728418b6a8237e591936c484e/dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractWorkflowResource.java#L489).

Reply:
I removed the following from `pom.xml`
 - `org.hibernate.common:hibernate-commons-annotations`
 - `jakarta.validation:jakarta.validation-api`
 
However, I found that the following are required,
 - `org.hibernate:hibernate-validator`
 - `org.glassfish:jakarta.el`
 
As even though it will compile, and the CLI will work normally, in certain circumstances you will get an error, for example if you execute `java -jar dockstore-client-1.13.0-SNAPSHOT-shaded.jar yaml validate --path /src/test/resources/YamlVerifyTestDirectory/2ToolsWithNoName` without the above required dependencies you will get this error,
```
HV000183: Unable to initialize 'javax.el.ExpressionFactory'. Check that you have the EL dependencies on the classpath, or use ParameterMessageInterpolator instead
at io.dockstore.client.cli.YamlVerify.dockstoreValidate(YamlVerify.java:211)
at io.dockstore.client.cli.YamlVerify.validateChecker(YamlVerify.java:226)
at io.dockstore.client.cli.YamlVerify.processEntryCommands(YamlVerify.java:103)
at io.dockstore.client.cli.Client.run(Client.java:733)
at io.dockstore.client.cli.Client.main(Client.java:630)
```